src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -115,6 +115,8 @@
 
     <!-- PGO assets by default are "Vertical" visibilty. Each repo will enable the specific artifacts it must publish externally -->
     <BuildArgs Condition="'$(PgoInstrument)' == 'true'">$(BuildArgs) /p:DefaultArtifactVisibility=Vertical</BuildArgs>
+    <!-- ShortStack builds only publish new assets from the root repository. All other assets are duplicates. -->
+    <BuildArgs Condition="'$(ShortStack)' == 'true' and '$(MSBuildProjectName)' != '$(RootRepo)'">$(BuildArgs) /p:DefaultArtifactVisibility=Vertical</BuildArgs>
```

Comment:
Short stack verticals build runtime but also its (transitive) dependencies. Those also publish. Is this fine because none of those produce RID specific assets?

Reply:
ok disregard. I read the above wrong.